src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -437,7 +437,12 @@ public static string GetInstanceId(this IEnvironment environment)
         {
             if (environment.IsAnyLinuxConsumption())
             {
-                return environment.GetEnvironmentVariableOrDefault(ContainerName, string.Empty);
```

Comment:
Can you explain this - seems unrelated to your changes.

Reply:
In flex, we want to replace CONTAINER_NAME with WEBSITE_POD_NAME. Bala's suggestion for getting the instance ID is to try and get WEBSITE_POD_NAME first and if not available fall back to CONTAINER_NAME. Both should be available right from container creation time and doesn't need specialization.